code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -1679,3 +1679,24 @@
 /datum/reagent/rmt/overdose(var/mob/living/carbon/H, var/alien)
 	if(prob(2))
 		to_chat(H, SPAN_WARNING(pick("Your muscles are stinging a bit.", "Your muscles ache.")))
+
+/datum/reagent/coagzolug
+	name = "Coagzolug"
+	description = "A medicine that was stumbled upon by accident, coagzolug encourages blood to clot and slow down bleeding. An overdose causes dangerous blood clots capable of harming the heart."
+	reagent_state = LIQUID
+	scannable = TRUE
+	color = "#bd5eb5"
+	overdose = 10
+	metabolism = REM / 3.33
+	taste_description = "throat-clenching sourness"
+	fallback_specific_heat = 1
+
+/datum/reagent/coagzolug/affect_blood(mob/living/carbon/M, alien, removed)
+	. = ..()
+	M.add_chemical_effect(CE_BLOODCLOT)
+	M.make_dizzy(5)
```

Comment:
How much dizziness does this stack in a 10u dose, exactly? This looks to metabolize gruelingly slowly and basically make you unable to walk right for hours.

Reply:
dizzy just makes the screen wobble